content/posts/android/Android Tips.md

```diff
@@ -144,17 +144,19 @@ GrapheneOS has the LTE only mode exposed in settings. You can set this by going
 
 If your Android-based operating system does not expose this setting in the Settings app, or if you want to set your baseband modem to a less restrictive mode, dial `*#*#4636#*#*` then hit **Phone information**. Here, you can set preferred network type to just the generations that you intend to use. For example, if you only want to use 5G and 4G, you can set it to `NR/LTE`.
 
-Depending on the carrier, you may need to enable additional network types for Wifi calling to work. For example, Google Fi requires WCDMA for this feature. Thus, if you want 5G, 4G, and Wifi calling for Google Fi, you need to set the network type as `NR/LTE/WCDMA`.
-
 ## Carrier Tracking
 
-Carriers can track your coarse location via cell towers using the IMSI and IMEI broadcasted by your baseband modem. In order to avoid this type of tracking, you have to enable the airplane mode which would disable the baseband modem. In addition, you will also need to disable Wifi calling to avoid IP based tracking by the carrier, as its connection will not go through your VPN tunnel.
+Carriers can track your coarse location through various means. In order to avoid this type of tracking, you need to do the following:
+
+- Disable Wifi-Calling. Wifi-calling connections do not go through the VPN tunnel, and thus will reveal your Wifi IP address to the carrier.
+
+- Disable the SIMs/eSIMs in **Settings** → **Network & internet** → **SIMs**. On GrapheneOS, if you are using an eSIM, you will need to enable the privileged eSIM management app before doing so. With certain carriers, there will be an ePDG server defined which the operating system will connect to outside of a VPN tunnel. While unlikely, a malicious carrier can track a user by giving them a unique ePDG server. 
 
-I have seen several common suggestions in the privacy community to mitigate this problem which do not actually work:
+- Turn on Airplane mode. This will turn off the baseband modem and your phone will no longer broadcast the IMEI/IMSI to the nearby towers. Note that your phone can still talk to the tower regardless of whether a SIM card inside - this is why `911` calls work. Simply removing the SIM cards is not enough.
 
-- **Removing the SIM Card**: The baseband modem will continue to contact the cell towers with its IMEI to prepare for emergency calls. In fact, this is how you are able to call `911` even when you do not have a SIM card inserted.
+- Disable the eSIM management app after you have disabled all of the eSIMs. With certain carriers, the eSIM management app will connect to the provisioning server to check for eSIM updates. 
```

Comment:
Do you mean the _privileged eSIM management toggle_ in GrapheneOS?

- The GOS toggle is very different from disabling an app. Figure out which, if either, achieves the intended result.
- If disabling this app turns out to be impossible on stock OS, then there's no point saying anything other than what works in GOS.
- You need to say what exact Google app is the relevant app here.

Reply:
It just means the esim management app. I am not sure how it works on random android devices. I guess we can just frame this for GOS for now, what u think?

Reply:
I still consider the package name/ID required information before merging this.

Reply:
I don't, this is just gonna sit for eternity before I can pull the correct package ID, and its is different depending on the OS. Not everyone uses the Google one.